src/dump-handlers.js

```diff
@@ -37,14 +36,17 @@ exports.mysql = function * (url, filename) {
   yield exec('tar', ['-czf', filename, 'dump'])
   return filename
 }
-
+// Back up a PostgreSQL database in a file call dump.sql, create an archive and compress it
 exports.postgres = function * (url, filename) {
+  console.log('>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>')
+  console.log(url)
   filename = `${filename}.tar.gz`
   const dbname = url.pathname.replace('/', '')
   const [user, pass] = _.split(url.auth, ':')
-  yield fse.writeFile('.pgpass', `${url.host}:5432:${dbname}:${user}:${pass}`)
+  const [host, port] = _.split(url.host, ':')
```

Comment:
port might not be set if the user does not set it in its URL : you have to check for empty values for port, and replace it with default port value for PostgreSQL (=5432)